packages/components/modules/notifications/native/NotificationsList/MarkAllAsReadButton/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from 'react'
+
+import { Button } from '@baseapp-frontend/design-system/components/native/buttons'
+
+import { useNotificationsMarkAllAsRead } from '../../../common'
+import { MarkAllAsReadButtonProps } from './types'
+
+const MarkAllAsReadButton: FC<MarkAllAsReadButtonProps> = ({ refetch, disabled = false }) => {
+  const [commitMutation, isLoading] = useNotificationsMarkAllAsRead()
+  const markAsRead = () => {
+    commitMutation({
+      onCompleted: refetch,
```

Comment:
it would be cool if we can do an optimistic update instead of a refetch here, just to avoid triggering the NotificationsList suspense fallback. Do we have more time into this story? Maybe do under another one?

Reply:
Yeah i think it would be for another story

Reply:
yea, lets talk about that on a breakout and create the story @Ronan-Fernandes